Structure_Engine/Query/ReinforcementTransitionPoints.cs

```diff
@@ -45,14 +45,14 @@ public static partial class Query
         [Output("locations", "The locations of reinforcement transitions in the crossection.")]
         public static List<double> ReinforcementTransitionPoints(this ConcreteSection concreteSection, double tolerance = Tolerance.Distance)
         {
-            if (concreteSection == null || concreteSection.Reinforcement == null || concreteSection.Reinforcement.Count == 0)
+            if (concreteSection == null || concreteSection.RebarIntent == null || concreteSection.RebarIntent.BarReinforcement.Count == 0)
```

Comment:
Check the inner list for null as well `concreteSection.RebarIntent.BarReinforcement == null` before checking the count